Misc/NEWS.d/next/Core and Builtins/2024-06-28-18-34-49.gh-issue-119726.Fjv_Ab.rst

```diff
@@ -0,0 +1,2 @@
+Optimise code layout for calls to C functions from the JIT on AArch64.
```

Comment:
We prefer US English throughout the project:

```suggestion
Optimize code layout for calls to C functions from the JIT on AArch64.
```

Reply:
Ok, it makes sense. Thanks for pointing that out.